modules/build/src/main/scala/scala/build/internal/ManifestJar.scala

```diff
@@ -0,0 +1,70 @@
+package scala.build.internal
+
+import java.io.OutputStream
+
+object ManifestJar {
+
+  def create(
```

Comment:
A scaladoc would be useful here, i.e. it wasn't immediately obvious to me what the `wrongSimplePaths` arg stands for

Reply:
Should be fixed.